### ISSUE TITLE
doc: CLIP_PADDING vs renderer.padding

### ISSUE THREAD

**Issue by rkjdid** [Leaflet/Leaflet#4132]
the doc mentions http://leafletjs.com/reference.html#path-clip_padding - a static L.CLIP_PADDING option for displaying out-of-viewport svg paths

for more recent releases, this option is replaced by `options.padding` in `L.Rendere` class - for instance, one would do at map initialization 
```
var map = L.map('map', {
      renderer: new L.SVG({
        padding: 1
      })
    });
```

**Comment by IvanSanchez** [comment]
@rkjdid Care to ellaborate a bit?

**Comment by rkjdid** [comment]
sure sorry, see edit

**Comment by IvanSanchez** [comment]
Depends on #3645.

**Comment by IvanSanchez** [comment]
Closed via #4203 / #3916
